tests/ltp/patches/ltp_fallocate_fallocate05_fix.patch

```diff
@@ -0,0 +1,66 @@
++ Test failure reason:
```

Comment:
Can you change the patch file name to fallocate05.patch
You can run 
git mv ltp_fallocate_fallocate05_fix.patch  fallocate05.patch